lint.js

```diff
@@ -80,10 +85,14 @@ More information: https://speccy.io/rules/#${rule.name}
 
 const command = async (file, cmd) => {
     const verbose = cmd.quiet ? 1 : cmd.verbose;
+    let rules = [].concat(cmd.rules);
 
     linter.initialize();
 
-    await loader.loadRuleFiles(cmd.rules, { verbose });
+    if(fs.existsSync(process.env["NODE_CONFIG_DIR"] + "/default.yml")) {
```

Comment:
This is what you were doing before, and we moved away from it. You dont need to check for the existance of a config file, just `let rules = [].concat(cmd.rules, config.get('lint.rules'));` and if there is no config file there will be no rules!

Reply:
I overlooked the `config.has()` method and incorrectly compensated for the resulting errors. That has been corrected.